app/src/ui/changes/sidebar.tsx

```diff
@@ -459,9 +459,19 @@ export class ChangesSidebar extends React.Component<IChangesSidebarProps, {}> {
           currentRepoRulesInfo={currentRepoRulesInfo}
           aheadBehind={this.props.aheadBehind}
           accounts={this.props.accounts}
-          filterText={this.props.changes.filterText}
+          filterText={this.props.changes.fileListFilter.filterText}
```

Comment:
We can just pass the "filterListFilter" object instead of splitting it all out here.